integrations/lambda/README.md

```diff
@@ -21,6 +21,7 @@ The following diagram depicts the deployed Lambda function receiving generated d
 ## Prerequisites
```

Comment:
This is a data generation utility and wondering why it is names "lambda" integration?. pls rename the project folder and move it under /tools

Reply:
I have changed the name of the application to "Timestream for LiveAnalytics Sample Data Generator," since it generates samples of real-life time series data. The generator is now in `tools/python/sample-data-generator/`.